### PR TITLE
Add SwissCovid work

### PR DESCRIPTION
**Pull request by ineiti**

### CODE REVIEW

File: data/SPRING/projects.yaml [hunk 1]
```diff
@@ -142,3 +142,31 @@ projects:
         date_updated: 2020-02-21
         maturity: 2
             
+    SwissCovid:
+        name: SwissCovid Exposure Notification
+        description: A privacy preserving exposure notification protocol to inform people about evenutal exposition to the CoVid-19 virus
```
Comment (cgrigis) [inline data/SPRING/projects.yaml:147]: I would not use the word "eventual" here (it is a "faux ami" with the French "éventuel"), unless you mean that all people will sooner or later be exposed. ;-)  → "about possible exposure"

Also, I think the official spelling is "COVID-19".

File: data/SPRING/projects.yaml [hunk 2]
```diff
@@ -142,3 +142,31 @@ projects:
...
+        layman_desc: >
+            The C4DT Factory team helped the SwissCovid effort by providing techincal specifications to the already written code.
```
Comment (cgrigis) [inline data/SPRING/projects.yaml:149]: `techincal` → `technical`

File: data/SPRING/projects.yaml [hunk 3]
```diff
@@ -142,3 +142,31 @@ projects:
...
+            In addition, we code reviewed, and helped Ubique to use github in a more Open Source respecting way.
```
Comment (cgrigis) [inline data/SPRING/projects.yaml:150]: → "we reviewed the code"

File: data/SPRING/projects.yaml [hunk 4]
```diff
@@ -142,3 +142,31 @@ projects:
         date_updated: 2020-02-21
         maturity: 2
             
+    SwissCovid:
+        name: SwissCovid Exposure Notification
```
Comment (cgrigis) [inline data/SPRING/projects.yaml:146]: "SwissCovid contact tracing"?

File: data/SPRING/projects.yaml (hunk 3 above)
Comment (ineiti) [reply data/SPRING/projects.yaml:150]: Oh my - I need to go out more and talk to actual people...

**Comment by ineiti** [comment]
Continued in #103 , sorry for merging too quickly...

**Pull request merged**